packages/web-components/src/components/multi-select/multi-select.scss

```
@@ -178,6 +186,17 @@ $css--plex: true !default;
   }
 }
 
+:host(#{$prefix}-multi-select[ai-label]) ::slotted(#{$prefix}-ai-label)::after {
+  position: absolute;
+  display: block;
+  background-color: $border-subtle-01;
+  block-size: convert.to-rem(16px);
```

Looks good, just a small thing here - we can use the spacing token
```suggestion
  block-size: $spacing-05;
```